wodles/aws/aws_s3.py

```diff
@@ -3112,6 +3135,9 @@ def get_script_arguments():
 def main(argv):
     # Parse arguments
     options = get_script_arguments()
+    if not tools.check_role_session_duration(options.role_session_duration):
```

Comment:
Perform this check in the suggested `arg_valid_iam_role_duration` function` function